bin/oref0-autotune-prep.js

```diff
@@ -87,6 +87,15 @@ if (!module.parent) {
         }
     }
 
+    // get insulin curve from pump profile that is maintained
+    profile_data.curve = pumpprofile_data.curve;
```

Comment:
Does the pumpprofile get updated when preferences.json is changed?

Reply:
It gets updated by oref0-pump-loop get_settings function by calling oref0-get-profile without the autotune profile.